### ISSUE TITLE
​All page list UI padding

### ISSUE THREAD

**Issue by himself65** [toeverything/AFFiNE#2933]
### What happened?

![image](https://github.com/toeverything/AFFiNE/assets/14026360/94e010f6-14b8-49e4-96db-8403b9cc87b2)

### Distribution version

macOS x64

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by tarunsamanta2k20** [comment]
I'd like to look into this. What is the issue. Could you please describe here?

**Comment by himself65** [comment]
> I'd like to look into this. What is the issue. Could you please describe here?

Please see https://www.figma.com/file/hVxJQtEqyv5xw1AbCzc50M/%F0%9F%97%92%EF%B8%8F-AFFiNE-2023?type=design&node-id=22-14415&mode=design&t=uyLvyoAcbBpFQGtL-4

Thank you for being so enthusiastic

**Comment by himself65** [comment]
What it should be like⬇️

![image](https://github.com/toeverything/AFFiNE/assets/14026360/09b44bc4-be1f-44c7-93b2-d2c2bc25d5c7)

**Comment by HuziiDev** [comment]
Just increase the width and height.

**Comment by Kaneki-Ken-7** [comment]
Hey @himself65 , I have solved this issue on my local. its just that I forked the repository and made changes in main branch so should I push it directly from main branch or make new branch then push it? I always get confused in version control problem , hope I could get some help here. 
I have attached the solved screenshot , those boxes are from the chrome extension. 
![All pages - AFFiNE](https://github.com/toeverything/AFFiNE/assets/30045569/ea2a715d-ec4e-4d2b-aaa4-b9e5786ab8e2)
